packages/devtools/lib/src/timeline/frames_bar_chart.dart

```diff
@@ -94,12 +95,12 @@ class FrameBar extends CoreElement {
     final gpuBarHeight =
         math.min(maxBarHeight - cpuBarHeight, frame.gpuDurationMs * pxPerMs);
 
-    final cpuTooltip = frame.isCpuSlow
-        ? _slowFrameWarning('GPU', frame.gpuAsMs)
-        : 'GPU: ${frame.gpuAsMs}';
-    final gpuTooltip = frame.isCpuSlow
-        ? _slowFrameWarning('GPU', frame.gpuAsMs)
-        : 'GPU: ${frame.gpuAsMs}';
+    final cpuMs = microsAsMsText(frame.cpuDuration);
```

Comment:
there's a cpuDurationMs / gpuDurationMs reason why we don't use this to convert to text (instead of dividing again to convert from micro to milli)?

Reply:
was trying to avoid re-creating ms text everywhere. Added another helper method that takes milliseconds.